play.pokemonshowdown.com/src/panel-chat.tsx

```diff
@@ -314,6 +314,7 @@ export class ChatRoom extends PSRoom {
 		'reject'(target) {
 			this.challenged = null;
 			this.update(null);
+			this.sendDirect(`/reject ${this.pmTarget || ''}`);
```

Comment:
Actually. Is this even necessary? `/reject` is already treated correctly by the server.